### PR TITLE
issue_110 updated scripts to read from environment variables

### PR DESCRIPTION
**Pull request by zqian**
- [x] added .env_example for for environment variables
- [x] updated the sis_upload.rb and sis_set_url.rb to read environment variables
- [x] added an README file with docker instructions
- [x] added a docker-compose.yml file

### CODE REVIEW

File: SIS_import/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+services:
+  job:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    volumes:
+      - ${HOME}/secrets/canvas
```
Comment (jonespm) [inline SIS_import/docker-compose.yml:11]: It doesn't look like this volume is being used. Is this for a future fix?

File: SIS_import/sis_set_url.rb [hunk 2]
```diff
@@ -549,84 +549,22 @@ def update_MPathway_with_Canvas_url(outputDirectory)
...
-		else
-			# break
-		end
-
-		#increase count
-		count=count+1
-	end
-	return return_hash
+	Dotenv.load
+	@logger.info("ddd")
```
Comment (jonespm) [inline SIS_import/sis_set_url.rb:553]: What is this for? Should be more descriptive if you want to leave it in here. like "Loading Dotenv"

**Review by jonespm — APPROVED** [review]

File: SIS_import/docker-compose.yml (hunk 1 above)
Comment (zqian) [reply SIS_import/docker-compose.yml:11]: It is not used. I will remove it

File: SIS_import/sis_set_url.rb (hunk 2 above)
Comment (zqian) [reply SIS_import/sis_set_url.rb:553]: removed!

**Pull request merged**